lib/util/sdp/index.js

```diff
@@ -251,6 +264,36 @@ function setSimulcast(sdp, sdpFormat, trackIdsToAttributes) {
   })).concat('').join('\r\n');
 }
 
+/**
+ * Return a new SDP string after reverting simulcast for non vp8 sections in remote sdp.
+ * @param sdp - simulcast enabled local sdp
+ * @param sdpWithoutSimulcast - local sdp before simulcast was set
+ * @param remoteSdp - remote sdp
+ * @return {string} Updated SDP string
+ */
+function revertSimulcastForNonVP8MediaSections(sdp, sdpWithoutSimulcast, remoteSdp) {
+  const remoteMidToMediaSectionMap = createMidMapForSdp(remoteSdp);
+  const noSimMidToMediaSectionMap = createMidMapForSdp(sdpWithoutSimulcast);
```

Comment:
@syerrapragada Nitpick: rename to `remoteMidToMediaSections` and `localMidToMediaSectionsWithoutSimulcast`?